src/features/accounts/components/accounts-menu/add-account-modal.tsx

```diff
@@ -164,26 +181,31 @@ const createCards = [
     label: "Hardware Authenticator",
     title: "create new with hardware authenticator",
     onClickArg: AddAccountMethodTypes.createAuthenticator,
+    requires: "idstore",
   },
 ];
 
 function CreateAccountOptions({
   onAddMethodClick,
+  services,
 }: {
   onAddMethodClick: (method: AddAccountMethodTypes) => void;
+  services: Set<string>;
 }) {
   return (
     <VStack alignItems="flex-start" w="full">
-      {createCards.map((c, idx) => {
-        return (
-          <AddAccountCard
-            key={idx}
-            label={c.label}
-            title={c.title}
-            onClick={() => onAddMethodClick(c.onClickArg)}
-          />
-        );
-      })}
+      {createCards
+        .filter((c) => !c.requires || services.has(c.requires))
```

Comment:
Here's the visibility logic: Either don't have a requirement or have a satisfied requirement.